libs/blog/ad-banner/data-access/src/lib/state/ad-banner.store.ts

```diff
@@ -17,13 +17,12 @@ type AdBannerState = {
 const initialState: AdBannerState = {
   adBannerVisible: true,
   adBanner: {
-    title: 'Angular Meetup',
-    number: 12,
-    description: `Learn how to use Angular's defer block to improve perfomance`,
-    date: '1 September',
-    time: '6pm CET',
-    location: 'Warsaw & Online',
-    href: 'https://google.com',
+    title: 'FREE EBOOK: The Ultimate Guide to Angular Evolution',
+    title_mobile: 'Free ebook',
+    description: `Discover How Angular 18 Improves Efficiency, DX, UX, and Performance`,
+    description_mobile: 'Angular Evolution',
+    additional_info: 'Angular 18',
+    href: 'https://houseofangular.io/the-ultimate-guide-to-angular-evolution/?utm_source=angular.love&utm_medium=baner&utm_campaign=angular-evolution24',
   },
```

Comment:
Having regard to the feature purposes I decided to append the service. Of course I could avoid doing it (for now store without the service would be ok), but in my opinion it will be fruitful in the nearest feature